apify-api/openapi/components/schemas/request-queues/GetHeadAndLockResponse.yaml

```diff
@@ -11,7 +11,15 @@ properties:
         example: 1000
       queueModifiedAt:
         type: string
+        description: The modifiedAt the queue is updated whenever the queue is modified. Modifications include adding, updating, or removing requests, as well as locking or unlocking requests in the queue.
```

Comment:
There is probably an extra "the queue" in the first sentence if I'm reading this right. Same in `RequestQueue.yaml`.

Reply:
Ups, I had the same comment but I forgot to send it 😅